subworkflows/local/variant_calling/call_snv_sentieon.nf

```diff
@@ -33,23 +35,29 @@ workflow CALL_SNV_SENTIEON {
             ch_vcf_idx = ch_vcf.join( ch_index )
 
             SENTIEON_DNAMODELAPPLY ( ch_vcf_idx, fasta, fai, ml_model )
-            ch_vcf      = SENTIEON_DNAMODELAPPLY.out.vcf
-            ch_index    = SENTIEON_DNAMODELAPPLY.out.vcf_index
             ch_versions = ch_versions.mix(SENTIEON_DNAMODELAPPLY.out.versions.first())
-        }
 
-        ch_vcf.join(ch_index)
-            .map { meta,vcf,tbi -> return [vcf,tbi] }
-            .set { ch_vcf_idx }
+            BCF_FILTER_ONE ( SENTIEON_DNAMODELAPPLY.out.vcf )
+
+            BCF_FILTER_TWO ( BCF_FILTER_ONE.out.vcf )
+
+            TABIX_BCFTOOLS ( BCF_FILTER_TWO.out.vcf )
+        
+            ch_vcf.join(TABIX_BCFTOOLS.out.vcf_index)
```

Comment:
Put the index file into the channel named ch_index , the same as in line 30. So we keep consistency of index channel names when we have ml_model and not.

Close the if statement here!
Move the join function after this closure